autopilot/stim/sound/sounds.py

```diff
@@ -171,11 +288,25 @@ def init_sound(self):
             if self.channel is None:
                 # The table will be 1-dimensional for mono sound
                 self.table = np.random.uniform(-1, 1, self.nsamples)
+                
+                if self.highpass is not None:
```

Comment:
I like this, but I think we can make this more general by using transform.timeseries.Filter_IIR object and passing one of those or filter coefs. i'll draft something in a moment